x-pack/legacy/plugins/fleet/public/pages/agent_list/index.tsx

```diff
@@ -185,8 +185,11 @@ export const AgentListPage: React.FC<{}> = () => {
         defaultMessage: 'Policy',
       }),
       truncateText: true,
-      render: (policyId: string) =>
-        (policies.find(p => p.id === policyId) || ({} as Policy)).name || `Policy: ${policyId}`,
+      render: (policyId: string) => (
+        <ConnectedLink color="primary" path={`/policies/${policyId}`}>
+          {(policies.find(p => p.id === policyId) || ({} as Policy)).name || `Policy: ${policyId}`}
```

Comment:
I like to use typescript optional chaining for case like this
```suggestion
{policies.find(p => p.id === policyId)?.name || `Policy: ${policyId}`}
```